src/main/java/wolox/training/models/Book.java

```diff
@@ -63,6 +67,7 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
+        Preconditions.checkNotNull(author);
```

Comment:
For all texts field that are required, it should validate that is not an empty text